spec/spec_helper.rb

```diff
@@ -59,6 +59,10 @@ def photo_fixture_name
   @photo_fixture_name = File.join(File.dirname(__FILE__), "fixtures", "button.png")
 end
 
+def jwks_file_path
+  @jwks_file = "../../spec/fixtures/jwks.json"
```

Comment:
It creates a file for every client application that wants to use `private_key_jwt` for their `token_endpoint_auth_method`. I initially tried to store the entire JWKs string as a database field using the default serializer (`serialize jwk_file, JSON`) and `t.string jwk_file` but it said it was too large for my MySQL database. In retrospect, I guess I could try `t.text jwk_file`.